src/display-x11.cc

```diff
@@ -227,6 +230,29 @@ bool display_output_x11::shutdown() {
   return true;
 }
 
+inline void propagate_unconsumed_event(XEvent &ev, bool is_consumed) {
+  const uint32_t capture_mask = ButtonPressMask | ButtonReleaseMask | ButtonMotionMask;
+
+  // SAFETY: XEvent is a union and all event types this function gets called for
+  // share same alignment and accessed fields share same offsets.
+  if (!is_consumed) {
+    XUngrabPointer(display, ev.xmotion.time);
+
+    /* forward the event to the desktop window */
+    ev.xmotion.window = window.desktop;
+    ev.xmotion.x = ev.xmotion.x_root;
+    ev.xmotion.y = ev.xmotion.y_root;
+    XSendEvent(display, window.desktop, False, capture_mask, &ev);
+    if (ev.type == ButtonPress) {
+      XSetInputFocus(display, window.desktop, RevertToNone, ev.xmotion.time);
+    }
+  } else {
+    XGrabPointer(display, window.window, TRUE, capture_mask,
+      GrabModeAsync, GrabModeAsync, window.window, None, ev.xbutton.time);
+    XSetInputFocus(display, window.window, RevertToParent, ev.xmotion.time);
```

Comment:
Now if the event was consumed by conky, it will capture focus and grab pointer so that the previously focused app needs to be focused again - same as if they clicked on the desktop, but we don't want desktop to be focused.

Reply:
Removed all pointer grabbing, we don't need it _at all_ AFAICT